src/components/PatientsTable/PatientsTable.test.tsx

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import { render, screen } from '@testing-library/react'
+import { PatientsTable } from '.'
+
+describe('PatientsTable', () => {
+  it('renders element', () => {
+    const onDelete = vitest.fn()
+    const editRoute = '/patients/edit'
```

Comment:
This can be just declared inline. If it's not reference anywhere, it doesn't have to be complicated